omc3/optics_measurements/phase.py

```diff
@@ -191,7 +191,7 @@ def _get_square_data_frame(data, index):
 def write_special(meas_input, phase_advances, plane_tune, plane):
     # TODO REFACTOR AND SIMPLIFY
     accel = meas_input.accelerator
-    meas = phase_advances["MEAS"]
+    meas = phase_advances['free']["MEAS"]
```

Comment:
for consistency reasons, wouldn't it be simpler if either 
a) the `["QF"]` selection part in the call would then also be moved into the function, or 
b) the `['free']` part be moved in the fct call?
personally, i'd prefer b) as it keeps the fct more general..

Reply:
I agree with b